stories/person.stories.js

```diff
@@ -24,3 +24,32 @@ export default {
 export const person = () => html`
   <mgt-person person-query="me" show-name show-email></mgt-person>
 `;
+
+export const personCard_hover = () => html`
+  <mgt-person person-query="me" person-card="hover"></mgt-person>
+`;
+
+export const personCard_inheritDetails = () => html`
```

Comment:
I understand why you moved the person card stories to the person story, but I think we still need the person card stories. What if we have the same stories for both?

Not sure if it will work, but you can still keep the personCard.stories.js and then try to import it here and reexport the stories so they show up in both